src/Workleap.Extensions.MediatR.ApplicationInsights/Workleap.Extensions.MediatR.ApplicationInsights.csproj

```diff
@@ -16,7 +16,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
```

Comment:
We have to keep this in sync with MediatR otherwise the compilation fails. At least it's just for netstandard2.0